config/main.py

```diff
@@ -3534,6 +3534,34 @@ def delete(ctx):
 
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
+@config.group(cls=AbbreviationGroup)
+@click.pass_context
+def switch(ctx):
+    """switch related config"""
+    pass
+
+@switch.command('shell')
```

Comment:
Can we have more specific command , like "diag_shell" or "sdk_shell" ?

Reply:
ok, will update to sdk_shell